client/src/types/register-client-option.model.ts

```diff
@@ -27,5 +27,10 @@ export type RegisterClientHelpers = {
     confirm?: { value: string, action?: () => void }
   }) => void
 
+  markdownRenderer: {
+    textMarkdownToHTML: (textMarkdown: string) => void
```

Comment:
`    textMarkdownToHTML: (textMarkdown: string) => Promise<string>`

Reply:
Thanks I also updated the documentation according to the returned type.